packages/pigeon/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.0-experimental.0+1
```

Comment:
This is already a `+hotfix` version string so adding another + here is confusing. I'm not totally confident pub can handle multiple punctuation like this. Does `-experimental.1` work?

Reply:
`dry-run` works for it, I can do '.1' if you think it's safer.